web/styles/settings.css

```diff
@@ -87,7 +87,12 @@ h3,
 
 #change_password_modal,
 #change_email_modal {
-    max-width: 480px;
+    width: 34.2857em; /* 480px at 14px/em */
+    max-width: 90vw;
+}
+
+#change-email-form-input-email {
+    max-width: 90%;
```

Comment:
A percentage-based `max-width` doesn't really make sense. Could this not just be `width`?

Reply:
I wanted to leave it as the default width size but not let it overflow its container. If you think having it always be 90% is better (i.e. quite long at wide screen sizes) then I can do that.